ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -461,7 +474,7 @@ struct MultiDeviceHostStorage {
         OwnedBuffer get_buffer(int buffer_index) const {
```

Comment:
While we're changing code here, wouldn't it be better to return this as a `const OwnedBuffer&`? Maybe the caller doesn't need to copy it.

Reply:
done.
I've foudn a few more places.